home-api-tests/pom.xml

```diff
@@ -2,15 +2,313 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
     <parent>
         <artifactId>home</artifactId>
         <groupId>com.softserveinc.ita.homeproject</groupId>
         <version>${revision}</version>
     </parent>
+
     <modelVersion>4.0.0</modelVersion>
+    <packaging>jar</packaging>
+    <artifactId>home-api-tests</artifactId>
 
-    <packaging>pom</packaging>
+    <properties>
+        <surefire-version>3.0.0-M5</surefire-version>
+        <assertj-version>3.18.1</assertj-version>
+        <commons-lang3-version>3.11</commons-lang3-version>
+        <fabric8io.docker.maven.plugin.version>0.24.0</fabric8io.docker.maven.plugin.version>
+        <docker.image.jdk.version>openjdk:11-jdk-buster</docker.image.jdk.version>
+        <docker.image.postgres.version>postgres:latest</docker.image.postgres.version>
+        <postgres.container.name>db-postgres</postgres.container.name>
+        <postgres.internal.port>5432</postgres.internal.port>
+        <postgres.db>postgres</postgres.db>
+        <postgres.user>postgres</postgres.user>
+        <postgres.password>password</postgres.password>
+        <postgres.url>jdbc:postgresql://${postgres.container.name}:${postgres.internal.port}/${postgres.db}
+        </postgres.url>
+        <datasource.url.property>spring.datasource.url</datasource.url.property>
+        <home.application.jar.name>home-application</home.application.jar.name>
+        <home.application.jar.version>${revision}</home.application.jar.version>
+        <home.data.migration.jar.name>home-data-migration</home.data.migration.jar.name>
+        <home.data.migration.jar.version>${revision}</home.data.migration.jar.version>
+        <liquibase.container.name>liquibase</liquibase.container.name>
+        <liquibase.internal.port>8000</liquibase.internal.port>
+        <home.application.internal.port>8080</home.application.internal.port>
+        <docker.artifacts.path>${project.build.directory}/docker/artifacts</docker.artifacts.path>
+    </properties>
 
-    <artifactId>home-api-tests</artifactId>
+    <profiles>
+        <profile>
+            <id>docker</id>
```

Comment:
Let's rename this profile to `api-tests-infrastructure`

Reply:
Done